src/AnalyzeCli.cpp

```diff
@@ -80,8 +95,19 @@ static bool analyzeFile(Luau::Frontend& frontend, const char* name, ReportFormat
         return false;
     }
 
-    for (auto& error : cr.errors)
+    WorkspaceFileResolver* fileResolver = static_cast<WorkspaceFileResolver*>(frontend.fileResolver);
+    std::filesystem::path rootUriPath = fileResolver->rootUri.fsPath();
+
+    for (auto& error : cr.errors) 
+    {
+        auto path = fileResolver->resolveToRealPath(error.moduleName);
+
+        if (isIgnoredFile(rootUriPath, *path, ignoreGlobPatterns)) {
+            continue;
+        }
```

Comment:
Could we move the `isIgnoredFile` check into `reportError`? `fileResolver` is already passed to it, so you can still get the rootUriPath